background.Rmd

```diff
@@ -21,9 +21,10 @@ Secondary aims are to gain insight into the predictive performance of different
 
 The European Covid-19 Forecast Hub aims at providing forecasts in real time and evaluating systematically how reliable these forecasts are. Please keep the following in mind when interpreting forecasts:
 
-- Real-time forecasts of epidemics, and particularly of Covid-19, have proven challenging in the past. Case forecasts in particular can be difficult in the face of changing testing practices, delays and capacities. Deaths are a more lagged measure and thus can be easier to predict. In this context it is important that forecasts are always interpreted with the associated prediction intervals.
-- Reported data and short-term trends can be subject to uncertainty in particular around holiday periods, when testing behaviour can be very difficult to predict across countries. Models may not reflect how this affects observed data and could misintepret a transient aberration in the data as a genuine trend.
+- Real-time forecasts of epidemics, and particularly of Covid-19, have proven challenging in the past. Case forecasts in particular can be difficult in the face of changing policies, population behaviour and testing practices. Deaths are a more lagged measure and thus can be easier to predict. Forecasts should therefore be always interpreted in conjunction with the associated prediction intervals. While these may sometimes seem wide, they contain detailed information on how likely different possible future trajectories are considered to be.
```

Comment:
can we remove "therefore" or remove the sentence about deaths being easier to predict? At the moment I think the logic doesn't really follow.